### PR TITLE
Added support for responsive layouts on capture/v2 

### PR DESCRIPTION
**Pull request by waridrox**
_Part of #645_ 
Make sure these boxes are checked before your pull request is ready to be reviewed and merged. Thanks!

* [ ] tests pass -- `rake test`
* [ ] code is in uniquely-named feature branch, and has been rebased on top of latest master (especially if you've been asked to make additional changes)
* [ ] pull request are descriptively named
* [ ] if possible, multiple commits squashed if they're smaller changes
* [ ] reviewed/confirmed/tested by another contributor or maintainer

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/wiki/contributing-to-public-lab-software

Thanks!

### CODE REVIEW

**Comment by RuthNjeri** [comment]
Hi @waridrox, is there a gif or an image to show the before and after with these changes? It would help in understanding the context. Also, is there an issue that this PR can reference which explains what this issue is fixing?

**Comment by waridrox** [comment]
Sure! 😅 But this should pe taken care of after the capture integration...

Current behaviour for different screen sizes - 

https://user-images.githubusercontent.com/58583793/123550437-118fe100-d78b-11eb-8e58-2debd7547cae.mp4


Proposed behaviour - 

https://user-images.githubusercontent.com/58583793/123550439-1359a480-d78b-11eb-9f57-b1057ef63203.mp4

**Comment by waridrox** [comment]
Hi could someone pls review this...
PS - I'm not good with css :P

File: app/assets/stylesheets/new-capture.css [hunk 1]
```diff
@@ -0,0 +1,570 @@
+html {
```
Comment (jywarren) [inline app/assets/stylesheets/new-capture.css:1]: Is this directly from the https://github.com/publiclab/spectral-workbench.js/ library? Could we pull its CSS in via yarn instead? And, i wonder, should the responsiveness changes happen over in that library as a result?

Comment (waridrox) [reply app/assets/stylesheets/new-capture.css:1]: Hmm, great idea ✨, should I try this in the `spectral-workbench.js` repo instead. Could this also mean that we need to update the sw-js lib the on `npm` and `yarn` dependencies...?

**Comment by jywarren** [comment]
If we've set version to ~0.2.0 or ^0.2.0 then the semver syntax will
resolve to the highest minor version number. This means a new version at
0.2.1 or 0.3.0 will fit the range we specified and it'll pick it up as long
as it's published on npm.

That said I think dependabot has the max # of PRS open already so it won't
open a new PR automatically for the new version unless we merge some of its
PRS. But we can run yarn install ourselves and it'll grab the new version
with an updated yarn.lock and we can check that file in.

Make sense?

**Comment by waridrox** [comment]
Sure got it! 😃

**Pull request closed without merging**